contents/handbook/company/management.md

```diff
@@ -27,16 +27,18 @@ This guidance applies to all teams, irrespective of whether you manage an engine
 
 ### Part-time managers
 
-Because of the relatively short list of tasks that managers have, management at PostHog is a part-time job. That means everyone, including the founders, still spend the majority of their time on practising what they do best - for most managers, this isn't actually management!
+Because of the relatively short list of tasks that managers have, management at PostHog is a part-time job. That means nearly everyone, oftentimes including the founders, still spend the majority of their time on practising what they do best - for most managers, this isn't actually management!
```

Comment:
```suggestion
Because of the relatively short list of tasks that managers have, management at PostHog is a part-time job. That means nearly everyone still spends the majority of their time on practising what they do best. For most managers, this isn't actually management!
```